docker/README.md

```diff
@@ -51,13 +51,15 @@ Simple stand-alone usage:
 
 To link to another mysql container, for example:
 
-    docker run --rm --name newcentury_mysql -p 3306 \
+    docker run --rm --name mysql \
         -e HUGINN_MYSQL_DATABASE=huginn \
         -e HUGINN_MYSQL_USER=huginn \
         -e HUGINN_MYSQL_PASSWORD=somethingsecret \
         -e HUGINN_MYSQL_ROOT_PASSWORD=somethingevenmoresecret \
         cantino/huginn
-    docker run --rm --name huginn --link newcentury_mysql:MYSQL -p 5000:5000 \
+    docker run --rm --name huginn \
+        --link mysql \
+        -p 5000:5000 \
         -e HUGINN_DATABASE_NAME=huginn \
         -e HUGINN_DATABASE_USER=huginn \
         -e HUGINN_DATABASE_PASSWORD=somethingsecret \
```

Comment:
The `HUGINN_DATABASE_PASSWORD` is being used to override the `DATABASE_PASSWORD`
The `HUGINN_DATABASE_USERNAME` should be used to override the DATABASE_USERNAME`

I think it is defaulting the `DATABASE_USERNAME` to be `root`.

I also think you are specifying `HUGINN_DATABASE_USER` which would be overriding something called `DATABASE_USER`, which isn't used anywhere.

I've comitted a fix for the README.md to reflect this.